myScript.js

```diff
@@ -39,4 +39,149 @@ function createList() {
   };
 }
 
-document.getElementById('humberger-icon').onclick = createList; // modify
+document.getElementById('humberger-icon').onclick = createList;
+
+/// /////////////////////////////////////////////////////////////////////
+
+const projects = [
+  {
+    title: 'Multi Post Stories',
+    discription: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    featureImg: './img/card1-project-img.png',
+    technologies: ['html', 'bootstrap', 'ruby on rail'],
+    linkToLive: 'https://github.com/',
+    linkToSource: 'https://github.com/yasinabdmahmood/portfolio-website',
+
+  },
+  {
+    title: 'Multi Post Stories',
+    discription: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    featureImg: './img/card1-project-img.png',
+    technologies: ['html', 'bootstrap', 'ruby on rail'],
+    linkToLive: 'https://github.com/',
+    linkToSource: 'https://github.com/yasinabdmahmood/portfolio-website',
+
+  },
+  {
+    title: 'Multi Post Stories',
+    discription: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    featureImg: './img/card1-project-img.png',
+    technologies: ['html', 'bootstrap', 'ruby on rail'],
+    linkToLive: 'https://github.com/',
+    linkToSource: 'https://github.com/yasinabdmahmood/portfolio-website',
+
+  },
+  {
+    title: 'Multi Post Stories',
+    discription: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    featureImg: './img/card1-project-img.png',
+    technologies: ['html', 'bootstrap', 'ruby on rail'],
+    linkToLive: 'https://github.com/',
```

Comment:
- Please on your mobile view popup, i notice some changes that should be fixed such as:
  - The buttons are not contained in the popup card container, it should be inside the card comntainer as in the figma design
  - PLease also work on spacing between elements, so your design looks a close as the figma design
##### Your design
![Screenshot from 2022-07-26 21-18-25](https://user-images.githubusercontent.com/58771507/181105026-bbcbc210-1886-4e03-bef3-1ee235ff60e4.png)

##### Figma design


![Screenshot from 2022-06-24 17-28-37](https://user-images.githubusercontent.com/58771507/181105111-f81c1d00-2218-442f-976d-03865cb63d72.png)